app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
     if params[:user][:search]
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/AlignHash
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L15

Align the elements of a hash literal if they span more than one line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*